packages/webpack/test/scaffold.js

```diff
@@ -113,7 +113,10 @@ function runScript(code, globals = defaultGlobals()) {
     throw new Error('runScript requires a bundle string as the first argument')
   }
   const context = createContext(globals)
-  return runInNewContext(code, context)
+  return {
+    context,
+    result: runInNewContext(code, context),
```

Comment:
refactoring this requires an update to all tests now.

Reply:
no one's using it 🎉